keras/layers/core.py

```diff
@@ -781,6 +778,9 @@ class Dense(Layer):
         nD tensor with shape: `(batch_size, ..., units)`.
         For instance, for a 2D input with shape `(batch_size, input_dim)`,
         the output would have shape `(batch_size, units)`.
+
+    Note: if the input to the layer has a rank greater than 2, the `kernel`
+    is applied to each slice along the last dimension of the input.
```

Comment:
I think this is a fairly confusing description, makes it sounds like distribution is along the last axis. Please rephrase.